content/1.7/back-office/catalog/brands-and-suppliers/brands-and-suppliers-brands.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Brands and suppliers brands
+weight: 1
+---
+
+In this page Brands are addresses are displayed.
+
+### Call to action
+
+2 call to actions are displayed in the header of this page:
+
+ 1) Add new brand
+
+👉 See the [specs of the add/edit brand]({{<ref "add-edit-brand.md">}} "Add/edit brand") 
+
+ 3) Add new brand address
+
+👉 See the [specs of the add/edit address]({{<ref "add-edit-address.md">}} "Add/edit address") 
+
+### Settings wheel
+
+When clicking on the settings wheel, the following options are displayed:
+1) Import
+
+After clicking on it, the Advanced parameters > Import page is opened.
```

Comment:
the  [Import page]({{<ref "../../advanced-parameters/import/">}} "Import Page")  is opened.